algorithms.tex

```diff
@@ -1,2 +1,17 @@
+% A description of some of the algorithms in SymPy. The list is not
```

Comment:
Should Algs be with the description of the respective feature instead of in a stand alone section?